File: cs-many-resources/Dummy.cs
using Pulumi;

public class Dummy : ComponentResource
{
    public Output<string> Deadweight { get; private set; } = null!;

    public Dummy(string name, Input<string> deadweight, ComponentResourceOptions? options = null)
        : base("examples:dummy:Dummy", name, options)
    {
        this.Deadweight = deadweight;
    }
}

File: ts-many-resources/dummy.ts
import * as pulumi from "@pulumi/pulumi";

export class Dummy extends pulumi.ComponentResource {
    public deadweight: pulumi.Output<string>;

    constructor(name: string, deadweight: pulumi.Input<string>, opts: pulumi.ComponentResourceOptions = {}) {
        super("examples:dummy:Dummy", name, {'deadweight': deadweight}, opts);

        this.deadweight = pulumi.Output.create(deadweight);
    }
}
